test/verify/check-testlib

```diff
@@ -128,12 +128,12 @@ class TestRunTest(MachineCase):
         process = subprocess.run([run_tests, "--test-dir", VERIFY_DIR, "TestExample.testFail", "TestExample.testSkip"],
                                  env=env, capture_output=True)
         stdout = process.stdout
-        self.assertRegex(stdout, rb"^not ok 1 .*test/verify/check-example TestExample.testFail # RETRY 1 \(be robust against unstable tests\)$")
```

Comment:
My first instinct is that this can't work without `re.M` multi-line matching. This has always been intended as a substring match.